docs/_pages/contributing.cn.md

```diff
@@ -25,7 +25,7 @@ To contribute code, you need
 
 To contribute, you would need to do the following
  
- - Fork the Hudi code on Github & then clone your own fork locally. Once cloned, we recommend building as per instructions on [quickstart](/docs/quick-start-guide.html)
+ - Fork the Hudi code on Github & then clone your own fork locally. Once cloned, we recommend building as per instructions on [quickstart](/docs/spark_quick-start-guide.html)
```

Comment:
`quickstart` -> `quickstart(Spark)`?

In addition, can we also add the link to flink guide?